app/main.py

```diff
@@ -1,3 +1,60 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    @staticmethod
+    def validate(other: Union[int, float, "Distance"]) -> float:
```

Comment:
From python 3.10+ we can use `|` for annotations. Fix in all cases

```suggestion
    def validate(other: int | float | Distance) -> float:
```